hw_pandas/Hw_pandas_pain.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import pandas as pd
+import re
+import seaborn as sns
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties
+import matplotlib.ticker as tck
+from matplotlib import rc,rcParams
+
+
+# Задание 1
+
+def read_gff(file):
+    gff_table = pd.read_csv(file, sep='\t', header = 0, names= ['chromosome', 
+                                                           'source', 'type',
+                                                           'start', 'end',
+                                                           'score', 'strand',
+                                                           'phase', 'RNA type'])
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на ммного строк, их нужно учитывать. Благо в пандасе это делается супер просто

```suggestion
    gff_table = pd.read_csv(file, sep='\t', comment="#", names= ['chromosome', 
                                                           'source', 'type',
                                                           'start', 'end',
                                                           'score', 'strand',
                                                           'phase', 'RNA type'])
```